cmd/gaia/app/genesis.go

```diff
@@ -124,7 +123,7 @@ func GaiaAppGenTxNF(cdc *wire.Codec, pk crypto.PubKey, addr sdk.Address, name st
 
 	validator = tmtypes.GenesisValidator{
 		PubKey: pk,
-		Power:  freeFermionVal,
+		Power:  freeDenomVal,
```

Comment:
`freeTokensVal` - denom mean denomination not quantity and denomination

Reply:
Sorry, I was still half asleep when I made that change. I'll change "denom" to "token" which makes much more sense.